be/src/exec/parquet_writer.h

```diff
@@ -35,13 +35,12 @@
 
 namespace starrocks {
 
-struct TableInfo;
-
 struct TableInfo {
     TCompressionType::type compress_type = TCompressionType::SNAPPY;
     bool enable_dictionary = true;
     std::string partition_location = "";
     std::shared_ptr<::parquet::schema::GroupNode> schema;
+    int64_t max_file_size = 1024 * 1024 * 1024; // 1GB
```

Comment:
why default value is 1GB?

Reply:
the default value in select into outfile and iceberg sink.